create.py

```diff
@@ -80,4 +82,12 @@ def logout():
         return render_template("reg.html", var1 = var1)
     except:
         var1 = "You must first log in to logout"
-        return render_template("reg.html",var1 = var1)
\ No newline at end of file
+        return render_template("reg.html",var1 = var1)
+
+
+@app.route('/books/<id>')
+def books(id):
+    result = db.session.query(Books).filter(Books.isbn == id)
```

Comment:
check if the user is logged in, if not do not display the page.

Reply:
Updated, checking user login.